build.gradle

```diff
@@ -1,11 +1,10 @@
 buildscript {
     repositories {
-        maven { url 'https://github.com/ben-manes/gradle-versions-plugin/raw/mvnrepo' }
         mavenCentral()
     }
 
     dependencies {
-        classpath 'com.github.ben-manes:gradle-versions-plugin:0.5-beta-5'
+        classpath 'com.github.ben-manes:gradle-versions-plugin:0.17.0'
```

Comment:
that doesn't seem to build ...

Reply:
Perhaps the 3.0 Gradle Wrapper is to old?

Works like a charm here: https://github.com/junit-team/junit5/blob/master/build.gradle#L7